docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build: .
     image: opbeans/opbeans-ruby:latest
     ports:
-      - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:5000"
+      - "127.0.0.1:${OPBEANS_SERVER_PORT:-8000}:3000"
```

Comment:
normalize port with other Opbeans

Reply:
Where are we setting the port at the ruby app? Because this is changing it at compose level, only

Reply:
❤️ . It's now aligned with the ITs:
- https://github.com/elastic/apm-integration-testing/blob/1f929e1f772e6513c919165ea0a8ce636b876ac8/scripts/modules/opbeans.py#L578

Reply:
Here it's where they port is configured: https://github.com/elastic/opbeans-ruby/search?q=5000&unscoped_q=5000

Reply:
But Puma is using the right port: https://github.com/elastic/opbeans-ruby/blob/6422d8642436a0992d9d1638414ddb6f9f8b8e89/config/puma.rb#L12